### PR TITLE
[mxnet] [test] [sagemaker] Security update to update vulnerable dependency urllib3

### PR DESCRIPTION
**Pull request by shantanutrip**
*Issue #, if available:*

## PR Checklist
- [x] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [ei/neuron] | [build] | [test] | [benchmark] | [ec2, ecs, eks, sagemaker]
- [ ] (If applicable) I've documented below the DLC image/dockerfile this relates to
- [x] (If applicable) I've documented below the tests I've run on the DLC image
- [ ] (If applicable) I've reviewed the licenses of updated and new binaries and their dependencies to make sure all licenses are on the Apache Software Foundation Third Party License Policy Category A or Category B license list.  See [https://www.apache.org/legal/resolved.html](https://www.apache.org/legal/resolved.html).
- [ ] (If applicable) I've scanned the updated and new binaries to make sure they do not have vulnerabilities associated with them.

## Reviewer Checklist
* For reviewer, before merging, please cross-check:
- [ ] I've verified the code change on the config file mentioned above has already been reverted

*Description:*
Older version of Urllib3 is a vulnerable dependency. This PR upgrades the version for Urllib3 to `urllib3>=1.26.5` . However, the new version of urllib3 was incompatible with the older version of `requests`.  Consequently, the version of`requests` had to be updated to 2.25.1.

*Tests run:*
SageMaker local integration tests were run on the following images:
1. beta-mxnet-training:1.8.0-cpu-py37-ubuntu16.04-benchmark-tested-2021-06-07-22-41-02
2. beta-mxnet-inference:1.8.0-cpu-py37-ubuntu16.04-benchmark-tested-2021-06-07-22-41-02
3. beta-mxnet-training:1.8.0-gpu-py37-cu110-ubuntu16.04-benchmark-tested-2021-06-07-22-41-02

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: test/sagemaker_tests/mxnet/training/requirements.txt [hunk 1]
```diff
@@ -5,8 +5,8 @@ pytest==5.3.5
 pytest-cov
 pytest-rerunfailures==9.0
 pytest-xdist
-requests==2.24
-urllib3>=1.25.10,<1.26.0
+requests==2.25.1 
+urllib3>=1.26.5
```
Comment (jeet4320) [inline test/sagemaker_tests/mxnet/training/requirements.txt:9]: latest it 1.26.6, so we can do `>=1.26.6`

Comment (shantanutrip) [reply test/sagemaker_tests/mxnet/training/requirements.txt]: done

**Comment by junpuf** [comment]
We can merge this after CI green

**Review by junpuf — APPROVED** [review]
LGTM

**Pull request merged**